api/tests/Data/Tests/Core/ContainerClients/RaceResultContainerClientTests.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Internal.RaceResults.Data.Utils;
+using Microsoft.Azure.Cosmos;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using RaceResults.Common.Models;
+using RaceResults.Data.Core;
+
+namespace Internal.RaceResults.Data.Core
+{
+    [TestClass]
+    public class RaceResultContainerClientTests
```

Comment:
since you have different data in these methods, another option is to
1. Create a method `private List<RaceResult> GenerateTesData(List<Guid> memerIds)` that returns `includedData`
2. Create a method `private RaceResult CreateRaceResult(Guid memberId)` that the above uses
3. Create a method `private void InitializeContainers(List<RaceResult> data)` that each method calls at the start

Reply:
Unified the data